tests/PHPStan/Rules/Classes/data/sealed.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Sealed;
+
+/**
+ * @phpstan-sealed FooClass|BarClass
+ */
+class BaseClass {}
+class FooClass extends BaseClass {}
+class BarClass extends BaseClass {}
```

Comment:
```suggestion
class BarClass extends BaseClass {}
class BarClassChild extends BarClass {}
```

Should be tested for all cases and pass.

Reply:
Added